bootloader.ld

```diff
@@ -35,8 +35,7 @@ SEARCH_DIR(.)
 /* Memory Spaces Definitions */
 MEMORY
 {
-  /* Reserve (at least) 32 bytes of space for factory install random bytes */
-  rom      (rx)  : ORIGIN = 0x00000000, LENGTH = 0x0000DFE0
+  rom      (rx)  : ORIGIN = 0x00000000, LENGTH = 0x0000E000

```

Comment:
the previous value was correct - the last 32 bytes are reserved for the factory randomness:

https://github.com/digitalbitbox/bitbox02-firmware/blob/1bcb6dd934635a454d5ddb13524da569ec5ea1e3/src/random.c#L96

That should maybe be put into its own definition in flags.h.

Please update the diagram too (reduce bootloader area by 32 bytes, add the 32 bytes factory randomness item).

Reply:
Interesting! I'll update it